mkdocs.yml

```diff
@@ -13,39 +13,60 @@ markdown_extensions:
-    - 'View Logs with Pimp my Log': 'extras/pimpmylog.md'
-    - 'Profile Code - XHProf, Blackfire, Xdebug': 'extras/profile-code.md'
-    - 'Debug Code with XDebug': 'extras/xdebug.md'
-    - 'Catch Emails with MailHog': 'extras/mailhog.md'
-    - 'Test with Behat and Selenium': 'extras/behat.md'
-    - 'Add Pre- and Post-Provision Scripts': 'extras/scripts.md'
+  - Getting Started:
+    - Installation:
+      - 'macOS': 'getting-started/installation-macos.md'
+      - 'Windows': 'getting-started/installation-windows.md'
+      - 'Linux': 'getting-started/installation-linux.md'
+    - 'Configuring Drupal VM': 'getting-started/configure-drupalvm.md'
+    - 'Syncing Folders': 'getting-started/syncing-folders.md'
+  - Building your codebase:
+    - 'Using a local Drupal codebase': 'deployment/local-codebase.md'
+    - 'Using a composer package': 'deployment/composer-package.md'
+    - 'Using composer.json': 'deployment/composer.md'
+    - 'Using a Drush Make file': 'deployment/drush-make.md'
+    - 'Drupal VM as a Composer Dependency': 'deployment/composer-dependency.md'
+  - Basic configurations:
+    - 'Using different base OSes': 'configurations/base-os.md'
+    - 'Using a different PHP version': 'configurations/php.md'
+    - Webservers:
+      - 'Apache': 'configurations/webservers-apache.md'
+      - 'Nginx': 'configurations/webservers-nginx.md'
+    - Databases:
+      - 'Connect to the MySQL Database': 'configurations/databases-mysql.md'
+      - 'Use MariaDB instead of MySQL': 'configurations/databases-mariadb.md'
+      - 'Use PostgreSQL instead of MySQL': 'configurations/databases-postgresql.md'
+  - Installed extras:
+    - 'adminer <small>(Database UI)</small>': 'extras/adminer.md'
+    - 'blackfire <small>(Profiling tool)</small>': 'extras/blackfire.md'
+    - 'drupalconsole <small>(Drupal CLI)</small>': 'extras/drupal-console.md'
+    - 'drush <small>(Drupal CLI)</small>': 'extras/drush.md'
+    - 'elasticsearch <small>(Search engine)</small>': 'extras/elasticsearch.md'
+    - 'java': 'extras/java.md'
+    - 'mailhog <small>(Mail catcher)</small>': 'extras/mailhog.md'
+    - 'memcached <small>(In-memory cache)</small>': 'extras/memcached.md'
+    - 'newrelic <small>(Performance monitoring)</small>': 'extras/newrelic.md'
+    - 'nodejs': 'extras/nodejs.md'
+    - 'pimpmylog <small>(Log viewer)</small>': 'extras/pimpmylog.md'
+    - 'redis <small>(In-memory database)</small>': 'extras/redis.md'
+    - 'ruby': 'extras/ruby.md'
+    - 'selenium <small>(BDD with Behat)</small>': 'extras/selenium.md'
+    - 'solr <small>(Search engine)</small>': 'extras/solr.md'
+    - 'tidways <small>(Profiling tool)</small>': 'extras/tideways.md'
+    - 'upload-progress': 'extras/upload-progress.md'
+    - 'varnish <small>(Caching reverse proxy)</small>': 'extras/varnish.md'
+    - 'xdebug <small>(Debugging tool)</small>': 'extras/xdebug.md'
+    - 'xhprof <small>(Profiling tool)</small>': 'extras/xhprof.md'
```

Comment:
Personal styling preference, this makes the name stand out while still explaining what it is.